### PR TITLE
Delete XR data exports & simplify export task

### PR DESCRIPTION
**Pull request by duncanjbrown**
## Context

These aren't used any more in the external report. This is because we derive the CSV downloads from the JSON (https://github.com/DFE-Digital/apply-for-teacher-training/blob/main/app/controllers/publications/monthly_statistics_controller.rb#L14-L21), so there's only a single source of truth for each external report.

## Changes proposed in this pull request

🚮

## Guidance to review

All gone?

## Link to Trello card

https://trello.com/c/mpp02RdH/4185-epic-make-monthly-report-production-ready

## Things to check

- [X] If the code removes any existing feature flags, a data migration has also been added to delete the entry from the database
- [X] This code does not rely on migrations in the same Pull Request
- [X] If this code includes a migration adding or changing columns, it also backfills existing records for consistency
- [X] API release notes have been updated if necessary
- [X] Required environment variables have been updated [added to the Azure KeyVault](/docs/environment-variables.md#deploy-pipeline)

### CODE REVIEW

**Comment by duncanjbrown** [comment]
> **Database-level enum changes detected**
> 
> Please include a data migration for these attributes and values:
> 

I'll leave these so we don't break historical data

**Comment by duncanjbrown** [comment]
Rebased

**Comment by duncanjbrown** [comment]
Added a couple more commits to clean up the import/export process

File: lib/tasks/monthly_report.rake [hunk 1]
```diff
@@ -21,67 +21,11 @@ end
 
 desc 'Write the latest MonthlyStatisticsReport to JSON and a set of CSV files'
 task export_monthly_report: :environment do
-  stats = Publications::MonthlyStatistics::MonthlyStatisticsReport.last.statistics.to_json
-  File.write('monthly_report.json', stats)
-
-  exports = EXPORTS.map do |export|
```
Comment (stevehook) [inline lib/tasks/monthly_report.rake:27]: Can we also get rid of the `EXPORTS` array above?

**Review by stevehook — APPROVED** [review]
Looks good to me. Is the plan to deactivate/remove the `lock_external_report_to_december_2021` feature flag after this goes live?

File: lib/tasks/monthly_report.rake (hunk 1 above)
Comment (duncanjbrown) [reply lib/tasks/monthly_report.rake:27]: yes!

**Pull request merged**